### PR TITLE
add localisation for thankyouMessage

### PR DESCRIPTION
**Pull request by s1mahanty**
## Related Issue
[FORMS-3187](https://jira.corp.adobe.com/browse/FORMS-3187)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java [hunk 1]
```diff
@@ -84,6 +85,8 @@ public abstract class AbstractComponentImpl implements Component {
      */
     private String id;
 
+    protected I18n i18n = null;
```
Comment (nit23uec) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java:88]: pls move initialisation of i18n in this class from AbstractFormComponentImpl.

**Comment by rismehta** [comment]
We would also need to change the rest API, @ravise5 is doing the same

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java (hunk 1 above)
Comment (s1mahanty) [reply bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java:88]: i18n is initialized in the base class, as discussed offline.

**Comment by rismehta** [comment]
Can you add a small unit test case for localization ?

**Pull request merged**